sympy/calculus/tests/test_util.py

```diff
@@ -63,6 +63,55 @@ def test_not_empty_in():
         Union(Interval(-2, -1, True, False), Interval(2, oo))
 
 
+def test_periodicity():
+    x = Symbol('x')
+    y = Symbol('y')
+
+    assert periodicity(sin(2*x), x) == pi
+    assert periodicity((-2)*tan(4*x), x) == pi/4
+    assert periodicity(sin(x)**2, x) == 2*pi
```

Comment:
Ans is `pi`, right ? [wolframalpha ans](http://www.wolframalpha.com/input/?i=period+y%3D+sin(x)**2)